Model/ExtendedShipOrder.php

```diff
@@ -0,0 +1,81 @@
+<?php
+namespace SnowIO\ExtendedSalesRepositories\Api\Data;
+
+use Magento\Framework\Api\Filter;
+use Magento\Framework\Api\Search\FilterGroup;
+use Magento\Framework\Api\SearchCriteria;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Magento\Sales\Api\ShipOrderInterface;
+use SnowIO\ExtendedSalesRepositories\Api\ExtendedShipOrderInterface;
+
+
+class ExtendedShipOrder implements ExtendedShipOrderInterface
+{
+
+    private $shipOrder;
+    private $orderRepository;
+
+    public function __construct(OrderRepositoryInterface $orderRepository, ShipOrderInterface $shipOrder)
+    {
+        $this->orderRepository = $orderRepository;
+        $this->shipOrder = $shipOrder;
+    }
+
+
+    /**
+     * Creates new Shipment for given Order.
+     *
+     * @param string $orderIncrementId
+     * @param \Magento\Sales\Api\Data\ShipmentItemCreationInterface[] $items
+     * @param bool $notify
+     * @param bool $appendComment
+     * @param \Magento\Sales\Api\Data\ShipmentCommentCreationInterface|null $comment
+     * @param \Magento\Sales\Api\Data\ShipmentTrackCreationInterface[] $tracks
+     * @param \Magento\Sales\Api\Data\ShipmentPackageCreationInterface[] $packages
+     * @param \Magento\Sales\Api\Data\ShipmentCreationArgumentsInterface|null $arguments
+     * @return int Id of created Shipment.
+     */
```

Comment:
This doc block can be removed as it's the same as in the interface. When a class has no doc block, it's inherited from its super class. Just make sure the web API still works without a doc block on the concrete class.